src/providers/ipa/ipa_subdomains_ext_groups.c

```diff
@@ -852,7 +908,27 @@ static struct tevent_req *ipa_add_trusted_memberships_send(TALLOC_CTX *mem_ctx,
     state->sdap_id_ctx = sdap_id_ctx;
     state->user_dn = user_dn;
     state->group_dom = group_dom;
-    state->groups = groups;
+    state->orig_groups = groups;
```

Comment:
What is the purpose to store `orig_groups`?

Reply:
Hi,

it is not strictly needed, but to underline that this array contains all strings we will use later on I added it to the state.

bye,
Sumit